process.md

```diff
@@ -0,0 +1,108 @@
+
+**4. Runtime Management**
+
+- **Track, Add, Remove Pallets** : Managing the components of the runtime environment to ensure optimal performance.
+- **Runtime Upgrades** : Establishing a timeline (TBD) and process for upgrading the network.
+- **Pallet Configuration Analysis** : Assessing whether new or existing pallet configurations align with Paseo's requirements.
+
+**5. Runtime Upgrade Testing and Coordination**
+
+- **Local Dry-Runs with Live State** : Testing upgrades in a controlled environment to ensure stability.
+- **Backporting** : Applying new features or fixes from the mainnet to the testnet.
+- **Bricked Parachains Resolution** : Developing strategies to deal with malfunctioning parachains.
+- **XCM HRMP Channels Management** : Handling cross-consensus message passing channels.
+- **XCM Debugging for Parachain Support** : Troubleshooting and supporting parachain-related issues.
+- **Parachain Slot Assignment:** Managing the allocation of slots to parachains.
+- **Benchmarking** : Comparing performance with Polkadot, taking into account different hardware configurations.
+- **Building Deterministic Runtime Artifacts** : Ensuring the runtime is predictable and repeatable.
+
+**6. Supported System Chains**
+Ensuring these specific chains are supported and integrated within Paseo.
+
+- Assethub
+- Bridgehub
+- People
+- Coretime
+- Collectives
+
+**7. Service Level Agreement (Support)**
+
+The workflow to acknowledge and resolve support tickets within specified time frames will be the following.
+
+- Support Ticket Handling
+  - **Acknowledgment** : Within 24 hours.
+  - **Resolution** : Mean time of 72 hours (_varies based on complexity and triage_)
+
+**8. High-Level Process Description**
+
+- Documentation Management:
+  - Wiki
+  - Processes
+  - Mini-Tutorials ([https://github.com/paseo-network/support](https://github.com/paseo-network/support))
+
+- Documentation Pull Requests
+  - **Review changes to documentation** : Regularly reviewing changes made to documentation ensures that updates align with current processes and standards, maintaining the integrity of the knowledge base.
+  - **Update Technical documentation** : Team members are responsible for keeping technical documentation up-to-date, reflecting the latest features, configurations, and troubleshooting steps.
+
+- Issue Management
+  - **Daily Support Triage** : Daily updates englobe communicating key information, achievements, or notable events to the public. This includes marketing efforts and maintaining effective communication over element, discord, X, etc.
+
+  - **Weekly Escalation Process:** Monthly reporting involves summarizing significant events, blog posts, or achievements. It provides a holistic view of the team's activities and contributions to Paseo.
+
+**8.1 Coordination Meetings**
+
+- Weekly every Tuesday: Core Team Meeting
+- Weekly every Thursdays: Internal Weekly Report
+- Monthly TBD: Fellowship Coordination
+
+**8.2 Reporting**
```

Comment:
I agree that some kind of reporting and community updates should happen, however a weekly report could result overkilling. Monthly feels ok and then we could of course include notifications in the middle for specific events such as runtime upgrades, issues communications etc.

wdyt?

Reply:
LGTM!, lets wait for other guys opinions also

Reply:
Agree with Hector, monthly feels more suitable.